### PR TITLE
Recipes Marketing Hat

### PR DESCRIPTION
**Pull request by leo-fsl**
https://bostoncommonpress.atlassian.net/browse/ACR-39

### CODE REVIEW

File: src/components/RecipesMarketingHat/RecipesMarketingHat.stories.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { action } from '@storybook/addon-actions';
+import AtkMarketingHat from './index';
```
Comment (jbonAA) [inline src/components/RecipesMarketingHat/RecipesMarketingHat.stories.js:3]: This story should import and use the new RecipesMarketingHat component

File: src/components/RecipesMarketingHat/index.js [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+};
+
+RecipesMarketingHat.defaultProps = {
+  className: null,
+  cta: 'Get Free Access',
+  description: 'Start your 2-week free trial and get instant access to everything.',
+  errorText: 'Invalid email address',
+  headline: 'Cook Anything with Confidence',
+  subheadline: 'Try these recipes',
+  heroImages: {
```
Comment (jbonAA) [inline src/components/RecipesMarketingHat/index.js:110]: Might be nice to use the new images as default props for this component

**Review by jbonAA — APPROVED** [review]

File: src/components/RecipesMarketingHat/index.js (hunk 2 above)
Comment (leo-fsl) [reply src/components/RecipesMarketingHat/index.js:110]: Solved

File: src/components/RecipesMarketingHat/RecipesMarketingHat.stories.js (hunk 1 above)
Comment (leo-fsl) [reply src/components/RecipesMarketingHat/RecipesMarketingHat.stories.js]: Solved

**Comment by atk-dm** [comment]
:rocket: PR was released in [`v1.33.2`](https://github.com/Americastestkitchen/mise-ui/releases/tag/v1.33.2) :rocket:

**Pull request merged**